src/app/shell/personal-cabinet/applications/applications.component.ts

```diff
@@ -27,21 +32,33 @@ import { ChildDeclination, WorkshopDeclination } from 'src/app/shared/enum/enumU
 })
 export class ApplicationsComponent extends CabinetDataComponent implements OnInit, AfterViewInit {
 
+  @Select(PaginatorState.applicationsPerPage)
+  applicationsPerPage$: Observable<number>;
+  @Select(UserState.applications)
+  applicationCards$: Observable<ApplicationCards>;
```

Comment:
there is already select in cabinet-data-component, so you  do not need to copy past it